docs-api/frontity-plugins/embedded-mode.md

```diff
@@ -0,0 +1,121 @@
+# Frontity Embedded Mode
+
+This plugin enables an **Embedded Mode** implementation of Frontity.
+
+- [Explore the code](https://github.com/frontity/frontity-embedded)
+- [Download the zip](https://github.com/frontity/frontity-embedded/archive/refs/heads/master.zip)
+
+Embedded Mode can be contrasted with **Decoupled Mode**. In Decoupled Mode the primary domain points to the Frontity site, with the WordPress site being on a secondary domain or on a subdomain of the primary domain.
+
+In Embedded Mode the primary domain points to the WordPress site, and the Frontity site can be on another domain - including the default URL provided by a free hosting service such as [Vercel](https://docs.frontity.org/deployment/deploy-using-vercel).
+
+This plugin replaces the active WordPress theme with the Frontity installation. Frontity therefore effectively becomes the WordPress sites' theme. It works by substituting it's own [`template.php`](https://github.com/frontity/frontity-embedded/blob/master/includes/template.php) in place of any call made to the [WordPress template hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy/).
+
+## Advantages of Embedded Mode
+
+Embedded Mode offers several advantages over Decoupled Mode.
+
+- The WordPress site exists under the primary domain, thus ensuring that:
+  - all the SEO benefits already built in to WordPress work as normal
+  - existing SEO (for a site transitioning to Frontity) will not be affected
+  - sitemaps generated by WordPress/Yoast/All-in-one-SEO work as normal
+  - WordPress page cache plugins and hosting CDNs continue to work as normal
+- Content producers/editors continue to have the same experience
+  - post/page preview remains available
+  - the admin bar is active for logged in users
+
+## Disdvantages of Embedded Mode
+
+- Frontity still needs to be hosted on a separate node server/serverless function (albeit on any domain you like including default domains provided by the hosting provider)
+- WordPress still needs to go through it's [bootstrap process](https://wordpress.tv/2017/06/22/alain-schlesser-demystifying-the-wordpress-bootstrap-process/) on initial page load
+- more routing is involved (WordPress makes a call to Frontity which returns the HTML to WordPress) so **a caching plugin is a necessity\*\*\*\*** rather than simply a nice to have. (We recommend [Simple Cache](https://github.com/luisherranz/simple-cache))
```

Comment:
```suggestion
- more routing is involved (WordPress makes a call to Frontity which returns the HTML to WordPress) so **a caching plugin is a necessity\*\*\*\*** rather than simply a nice to have.
```

I'd remove that for now because it doesn't work yet.